plugins/outputs/http/http.go

```diff
@@ -14,16 +14,16 @@ import (
 	"strings"
 	"time"
 
-	awsV2 "github.com/aws/aws-sdk-go-v2/aws"
-	v4 "github.com/aws/aws-sdk-go-v2/aws/signer/v4"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/aws/signer/v4"
```

Comment:
`v4` is just a very unclear name.
```suggestion
	aws_signer "github.com/aws/aws-sdk-go-v2/aws/signer/v4"
```

Reply:
Makes sense, changed.